### ISSUE TITLE
Build failed with `rust-safe 0.68.0 and 0.69.0` 

### ISSUE THREAD

**Issue by aimixsaka** [rust-skia/rust-skia#913]
## Description
I'm building [neovide 0.12.0](https://github.com/neovide/neovide/tree/0.12.0) (which depends on `rust-safe 0.68.0`) on Arch Linux RISC V.
- My compiling command:
```
export CXXFLAGS="${CXXFLAGS/_FORTIFY_SOURCE=3/_FORTIFY_SOURCE=2}"
CFLAGS+=' -ffat-lto-objects'
export SKIA_USE_SYSTEM_LIBRARIES=true
export FORCE_SKIA_BUILD=true
export SKIA_GN_COMMAND=gn
export SKIA_NINJA_COMMAND=ninja
cargo build --frozen --release --features embed-fonts
```

- Error Log:
```
...
--- stderr
  thread 'main' panicked at /build/.cargo/git/checkouts/rust-skia-627fca40798b1c0d/5bec56d/skia-bindings/build_support/skia_bindgen.rs:786:59:
  called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }
...
```

- Full error log: http://fars.ee/PFjl

- After dig into directory, I find it missing the `obj/third_party/icu/icu.ninja` file, which is declared by
  https://github.com/rust-skia/rust-skia/blob/f0bea3ad919500904bb9b27fcccf0a361b9056d9/skia-bindings/build_support/skia_bindgen.rs#L814C40-L814C40

- Same issue on x86_64
- Same issue on `master branch`

## How to reproduce directly on `rust-skia`
1. install `gn ninja git patch cargo python clang fontconfig freetype2 libjpeg cmake`
2.
```
export FORCE_SKIA_BUILD=true
export SKIA_USE_SYSTEM_LIBRARIES=true
export SKIA_NINJA_COMMAND=ninja
export SKIA_GN_COMMAND=gn
```
3.
```
git clone https://github.com/rust-skia/rust-skia --recurse-submodules
cd rust-skia

#  Prevent calling fetch-gn if gn is specified in env 
patch -Np1 -i <(curl -L https://github.com/aimixsaka/rust-skia/commit/cf7b521d631f8a8d1bcc82449c3e2bc8b14cecd4.diff)
# Make fetch-gn optional in git-sync-deps 
patch -d skia-bindings/skia/ -Np1 -i <(curl -L https://github.com/aimixsaka/skia/commit/00c0df064e0595deff2aa14fb73ee8a6d90b80b9.diff)

cargo build -vv -p skia-safe --features gl --features textlayout
```

**Comment by pragmatrix** [comment]
I suspect that the problem is that icu is not compiled when `SKIA_USE_SYSTEM_LIBRARIES=true`, but the icu.ninja file is expected to exist for reading out the declarations.

**Comment by pragmatrix** [comment]
Can you try to build with the branch in #916?

**Comment by aimixsaka** [comment]
> Can you try to build with the branch in #916?

Tested on both x86_64 and riscv64, truly fixed by #916. Thx :)
